features/upgrade/sdn/policy-upgrade.feature

```diff
@@ -0,0 +1,47 @@
+Feature: SDN compoment upgrade testing
+  # @author zzhao@redhat.com
+  @admin
+  @upgrade-prepare
+  Scenario: Check the networkpolicy works well after upgrade - prepare
+    Given I switch to cluster admin pseudo user		
```

Comment:
Put this before the step when cluster-admin is needed, instead of at the very beginning, or else all steps are executed via cluster-admin.

Reply:
this is expected, all steps will use admin user

Reply:
Do we need to be cluster-admin to create project, create pods, check pods are ready ???

Reply:
yes, this care do not care the user.  it just care the namespaces ID
so we use admin user for upgrade scenarios here. just for convenience from geliu advice.

Reply:
I agree running as `admin` makes sense but I just found a case where this can cover up some issues with other steps that don't use `admin` properly, see https://github.com/openshift/verification-tests/pull/944